NotificationServiceExtension/NotificationService.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  NotificationService.swift
+//  NotificationServiceExtension
+//
+//  Created by NamNH on 21/02/2022.
+//
+
+import UserNotifications
+
+class NotificationService: UNNotificationServiceExtension {
+	
+	var contentHandler: ((UNNotificationContent) -> Void)?
+	var bestAttemptContent: UNMutableNotificationContent?
+	
+	override func didReceive(_ request: UNNotificationRequest, withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
+		self.contentHandler = contentHandler
+		bestAttemptContent = (request.content.mutableCopy() as? UNMutableNotificationContent)
+		
+		if let bestAttemptContent = bestAttemptContent {
+			// Modify the notification content here...
+			bestAttemptContent.title = "\(bestAttemptContent.title) [modified]"
+			
+			contentHandler(bestAttemptContent)
+		}
+	}
+	
+	override func serviceExtensionTimeWillExpire() {
+		// Called just before the extension will be terminated by the system.
+		// Use this as an opportunity to deliver your "best attempt" at modified content, otherwise the original push payload will be used.
+		if let contentHandler = contentHandler, let bestAttemptContent =  bestAttemptContent {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/59ef417893024699d72190a85b69abcca879c633/NotificationServiceExtension/NotificationService.swift#L30"></span>Operators should be surrounded by a single whitespace when they are being used.<br />
<code>operator_usage_whitespace</code> <code>NotificationService.swift:30</code></td>
    </tr>
  </tbody>
</table>